src/Arcus.Templates.WebApi/appsettings.json

```diff
@@ -9,5 +9,10 @@
     }
   },
   //#endif
-  "MyConfigKey":  "MyConfigValue"
+  //#if (JwtAuth)
+  "Jwt": {
+    "Secret": "YOUR SECRET KEY"
```

Comment:
You validate the user right? why storing the secret (of I assume the user) in the config?

Reply:
Sorry, it seems to be the signingkey. Then I would rename this property